### PR TITLE
client: enhance flush ack  log

### PR DESCRIPTION
**Pull request by taodd**

### CODE REVIEW

File: src/client/Client.cc [hunk 1]
```diff
@@ -4140,10 +4140,11 @@ int Client::mark_caps_flushing(Inode *in, ceph_tid_t* ptid)
   in->flushing_cap_tids[flush_tid] = flushing;
 
   if (!in->flushing_caps) {
-    ldout(cct, 10) << "mark_caps_flushing " << ccap_string(flushing) << " " << *in << dendl;
+    ldout(cct, 10) << __func__ << " newly flush " << ccap_string(flushing) << " " << *in << dendl;
     num_flushing_caps++;
   } else {
-    ldout(cct, 10) << "mark_caps_flushing (more) " << ccap_string(flushing) << " " << *in << dendl;
+    ldout(cct, 10) << __func__ << " newly flush " << ccap_string(flushing) 
+                   << " currently is flushing " << ccap_string(in->flushing_caps) << " " << *in << dendl;
```
Comment (ukernel) [inline src/client/Client.cc]: '<< *in ' already includes flushing_caps

Comment (taodd) [reply src/client/Client.cc]: thank you for your review
forgot to check "<< *in", 
i will rebase it

**Comment by taodd** [comment]
retest this please

**Comment by taodd** [comment]
retest this please

**Comment by taodd** [comment]
@batrick updated

**Pull request merged**